Apps/APN/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
This file looks like it was copied from the Ami app. I would delete it from this PR. The whole code base has a LICENSE file in the root directory already.

Reply:
> I would delete it from this PR

Did you delete the file ? I still see the file in this folder. Want to confirm before making any changes.

Reply:
No, this file still exists in branch `aman/apn-ios` and has not yet been deleted.